### DISCUSSION TITLE
Re add Os. Commands on lua

### DISCUSSION THREAD

**Discussion by MemTheBun** [DEMOLITIONDON96/Demolition-Engine#13] [Ideas (Softcode)]
YES DOIT

**Comment by ThatOneSillyGuyDon** [comment]
k, fine, only gonna add the non-malicious ones, cause i still don't trust it imo

**Comment by Blossomical** [comment]
no its dangerous

**Reply by djnbeeman** [reply]
demolition will only re-add the non-malicious ones.

**Comment by Theoyeah** [comment]
i know how to do it
